src/main/java/org/synyx/urlaubsverwaltung/absence/web/AbsenceOverviewViewController.java

```diff
@@ -142,35 +143,57 @@ public String absenceOverview(
         model.addAttribute("selectedMonth", selectedMonth);
 
         final List<Person> membersOfSignedInUser = getActiveMembersOfPerson(signedInUser);
-        model.addAttribute("showRichLegend", !membersOfSignedInUser.isEmpty());
+        final boolean isSignedInUserAllowedToSeeAbsences = !membersOfSignedInUser.isEmpty();
+        model.addAttribute("sickNoteLegendVisible", isSignedInUserAllowedToSeeAbsences);
 
-        // use active vacation types instead of all to avoid too much items in the legend.
+        final List<VacationType> vacationTypes = vacationTypeService.getAllVacationTypes();
+        final Map<Integer, VacationType> vacationTypesById = vacationTypes.stream().collect(toMap(VacationType::getId, Function.identity()));
+
+        // use active vacation types instead of all to avoid too many items in the legend.
         // (there could be non-active vacation types visible in the absence-overview)
         // the legend will be removed soon -> therefore display just the active items
-        final List<VacationType> vacationTypes = vacationTypeService.getActiveVacationTypes();
-        model.addAttribute("vacationTypeColors", vacationTypes.stream().map(AbsenceOverviewViewController::toVacationTypeColorsDto).collect(toUnmodifiableList()));
+        List<VacationTypeColorDto> vacationTypeColorDtos = prepareVacationTypeColorsForLegend(isSignedInUserAllowedToSeeAbsences, vacationTypes);
+        model.addAttribute("vacationTypeColors", vacationTypeColorDtos);
+
+        final Function<AbsencePeriod.RecordInfo, Boolean> shouldAnonymizeAbsenceType =
```

Comment:
Das ich auch meine sehen darf sollten wir hier doch 

```java
final Function<AbsencePeriod.RecordInfo, Boolean> shouldAnonymizeAbsenceType = recordInfo -> !recordInfo.getPerson().equals(signedInUser) && !membersOfSignedInUser.contains(recordInfo.getPerson()) && !recordInfo.isVisibleToEveryone();
```

Dann werden sie anonymisiert, wenn
* ICH es NICHT bin
* ich diese Person nicht "manage" (nicht als AL/SSA/... die Person in meiner Abteilung habe)
* der Abwesenheitstyp nicht für alle sichtbar ist

eins von denen eben zutrifft.

Reply:
Done